.circleci/config.yml

```diff
@@ -34,6 +32,22 @@ jobs:
         command: ./e2e-harness teardown
         when: always
 
+  e2eTestFrameworkIntegration:
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/operatorkit
+    steps:
+    - checkout
+
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./e2e-harness
+    - run: ./e2e-harness localkube
+    - run: ./e2e-harness setup --remote=false
+    - run: ./e2e-harness test --test-dir=framework/integration
```

Comment:
I would appreciate a separation here right away so we can have a more proper project structure and also add other tests we can run in parallel later on. Feel free to separate this in an upcoming PR before providing a PR for the other integration test.

Reply:
What do you mean by separation here? Do we have a project where we do this? I am not too familiar with this circle magic.

Reply:
See https://github.com/giantswarm/aws-operator/blob/master/.circleci/config.yml and https://github.com/giantswarm/aws-operator/tree/master/integration/test and so on.